src/main/java/JavaLessons/Lesson1/Lesson6/Cat.java

```diff
@@ -0,0 +1,29 @@
+package JavaLessons.Lesson1.Lesson6;
+
+public class Cat extends Animal {
+
+    protected static int catCount;
+
+    public Cat(String name, int runDistance, int swimDistance) {
+        super(name, runDistance, swimDistance);
+        catCount++;
+    }
+
+    public static int getCatCount() {
+        return catCount;
+    }
+
+    @Override
+    public void run() {
+        if (runDistance < 200) {
```

Comment:
Avoid using magic numbers. You could easily create a constant with 'speaking' name like 
```Java
private static final int CAT_MAX_RUN_DISTANTION_IN_METERS = 100500;
```
and everybody, including yourself would understant what that number means, because it is not obvious for others and for you from the future.